src/ui/units/dash/containers/Body/Body.tsx

```diff
@@ -310,35 +324,35 @@ class Body extends React.PureComponent<BodyProps, DashBodyState> {
         lastDelayedScrollTop: null,
         groups: {
             margins: DEFAULT_DASH_MARGINS,
-            renderers: [
-                {
-                    id: FIXED_GROUP_HEADER_ID,
-                    render: (id, children, props) =>
-                        this.renderFixedGroupHeader(
-                            id,
-                            children,
-                            props as DashkitGroupRenderWithContextProps,
-                        ),
-                    gridProperties: fixedHeaderGridProperties(),
-                },
-                {
-                    id: FIXED_GROUP_CONTAINER_ID,
-                    render: (id, children, props) =>
-                        this.renderFixedGroupContainer(
-                            id,
-                            children,
-                            props as DashkitGroupRenderWithContextProps,
-                        ),
-                    gridProperties: getPropertiesWithResizeHandles(),
-                },
-                {
-                    id: DEFAULT_GROUP,
-                    gridProperties: getPropertiesWithResizeHandles(),
-                },
-            ],
+            renderers: [],
         },
     };
 
+    constructor(props: BodyProps) {
+        super(props);
+
+        this.state.groups.renderers = [
```

Comment:
Please don't do it like this declare whole init state in constructor or as a property. It would be hard to find all initial state just by reading code in future